null_safety/calculate_lix/README.md

```diff
@@ -0,0 +1,79 @@
+# Null safety example: A CLI-app for calculating lix
+
+This is a small code example of an app that calculates the 'lix' readability
+index for a text file. The implementation uses the new Dart null safety feature,
+and is meant to demonstrate how this feature works in a practical example,
+as well as serve as a demonstration of how to configure and run code with null
+safety at it's current technical preview stage.
+
+# Running the example code
+
+The code works only with the first technical preview of null safety, Dart SDK
+version `2.9.0-13.0.dev`. You will need to download a copy of this Dart SDK even
+if you have a Flutter or Dart SDK installed already, and you'll want to use this
+preview SDK only for experimenting with null safety. Specifically, do not use it
+for any kind of production coding.
+
+## Dart preview SDK installation
+
+  1. Download the preview SDK:
```

Comment:
Better to point folks to https://dart.dev/tools/sdk/archive ?

Won't be fun to keep these links up-to-date

Reply:
I think we want these pinned to a known good version until we get to beta?

Reply:
I'd rather we encourage folks to get latest! If something breaks between here and there, we want folks to file issues, complain, etc...

Reply:
OK, compromise is that I'll link to SDK archive, but mention 2.9.0-13.0.dev as the TP1 build